src/main/java/org/tron/common/runtime/RuntimeImpl.java

```diff
@@ -97,6 +96,10 @@
   private TransactionTrace trace;
   private boolean isStaticCall;
 
+  // TODO according to version
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbTDq44O08i52vd9SHK&open=AWbTDq44O08i52vd9SHK)

[ISSUE_KEY]: AWbTDq44O08i52vd9SHK